packages/gestalt/src/Box.test.js

```diff
@@ -8,6 +8,11 @@ test('Box renders', () => {
   expect(tree).toMatchSnapshot();
 });
 
+test('Box has correct default display flex class', () => {
```

Comment:
This test is asserting the same thing as the test above it - perhaps testing explicitly setting the display property would be more useful.

Reply:
Thanks! Updated with more explicit tests.